scripts/docker-run-restore-points-test.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+SCRIPT_DIR=$(dirname $0)
+BASE_DIR=${PWD}/${SCRIPT_DIR}/..
+DO_CLEANUP=true
+
+while getopts "d" opt;
+do
+    case $opt in
+        d)
+            DO_CLEANUP=false
+            echo "!!Debug mode: Containers and temporary directory will be left on disk"
+            echo
+            ;;
+    esac
+done
+
+shift $((OPTIND-1))
+
+if "$DO_CLEANUP" = "true"; then
+    trap cleanup EXIT
+fi
+
+cleanup() {
+    # Save status here so that we can return the status of the last
+    # command in the script and not the last command of the cleanup
+    # function
+    status="$?"
+    set +e # do not exit immediately on failure in cleanup handler
+    # docker rm -vf timescaledb-valgrind 2>/dev/null
+    docker rm -vf timescaledb-rp 2>/dev/null
+    echo "Exit status is $status"
+    exit $status
+}
+
+docker_exec() {
+    # Echo to stderr
+    >&2 echo -e "\033[1m$1\033[0m: $2"
+    docker exec $1 /bin/bash -c "$2"
+}
+
+docker rm -f timescaledb-rp 2>/dev/null || true
+IMAGE_NAME=rp_test TAG_NAME=latest bash ${SCRIPT_DIR}/docker-build.sh
```

Comment:
I think we are now building this Docker image over-and-over again in different workflows, which seems inefficient. We should probably see if we can build it once, cache, and reuse across different tests.

If it is difficult to do, we can do it later but just noting the duplicate work here.

Reply:
But we build them against different PG version, not sure what we can do here